db/migrate/20220922080350_migrate_api_clients_to_api_keys.rb

```diff
@@ -0,0 +1,13 @@
+class MigrateApiClientsToApiKeys < ActiveRecord::Migration[6.1]
+  def change
+    files = Dir[Rails.root + 'app/models/*.rb']
+    models = files.map{ |m| File.basename(m, '.rb').camelize }
+
+    if models.include?("ApiClient")
+      ApiClient.all.each do |api_client|
```

Comment:
@Pralish we will remove `ApiClient` related code after merging this feature right?